src/main/docs/index.md

```diff
@@ -50,6 +50,17 @@ Here is the simplest way to connect, query and disconnect
 {@link examples.Examples#gettingStarted()}
 ```
 
+You can also use environment variables to set default connection setting values, this is useful
+when you want to avoid hard-coding database connection information. 
```

Comment:
I think we are missing a link to the libpg page that defines these env variables

Reply:
so maybe I need to add something like:
 > You can refer to the [official documentation](https://www.postgresql.org/docs/9.6/static/libpq-connect.html#LIBPQ-CONNSTRING) for more details.

this is ok?

Reply:
sounds good